packages/native-ui/src/components/List/ListHeading/ListHeading.props.ts

```diff
@@ -1,6 +1,6 @@
 import type { ViewProps } from 'react-native';
 
-interface ListHeadingBaseProps extends Omit<ViewProps, 'children'> {}
+type ListHeadingBaseProps = Omit<ViewProps, 'children'>;
```

Comment:
Triggers me mixing types and interfaces like this 😅

Reply:
aye. Just doing what the linter says 😆 we can always look at bypassing the rule?